module.txt

```diff
@@ -1,17 +1,25 @@
 {
     "id" : "AdditionalWorlds",
-    "version" : "0.2.0-SNAPSHOT",
+    "version" : "0.2.1-SNAPSHOT",
     "author" : "",
     "displayName" : "AdditionalWorlds",
     "description" : "",
     "dependencies" : [
+        {
+            "id" : "CoreAssets",
+            "minVersion" : "1.0.0"
+        },
+        {
+            "id" : "CoreBlocks",
+            "minVersion" : "1.0.0"
+        },
 		{
 			"id" : "Core",
-			"minVersion" : "2.0.0"
+			"minVersion" : "2.2.1"
```

Comment:
This might not have been visible on the diff locally, but see how on the PR here on GitHub the indentation between your additions and the existing ones differ? In this case you're doing it right and the existing stuff uses _evil tabs_ ! Oh noes. If you ever spot this please feel free to fix up the whitespace in the remainder of the file :-)